### ISSUE TITLE
Shared Library Issues on Linux

### ISSUE THREAD

**Issue by krystiancharubin** [kroo/wyzecam#10]
## 🐛 Bug Report

Code seems to always look for the .dylib (mac) version of the shared library even on Linux.
https://github.com/kroo/wyzecam/blob/9cf17fb9a3ccb4024c645efc0ecb542b28bef1e2/wyzecam/tutk/tutk.py#L508

## 🔬 How To Reproduce

1. Run any of the sample code in Linux

```bash
python3 wyze.py
Traceback (most recent call last):
  File "wyze.py", line 10, in <module>
    with wyzecam.WyzeIOTC() as wyze_iotc:
  File "/home/kcharubin/.local/lib/python3.8/site-packages/wyzecam/iotc.py", line 74, in __init__
    tutk_platform_lib = tutk.load_library()
  File "/home/kcharubin/.local/lib/python3.8/site-packages/wyzecam/tutk/tutk.py", line 515, in load_library
    return cdll.LoadLibrary(shared_lib_path)
  File "/usr/lib/python3.8/ctypes/__init__.py", line 451, in LoadLibrary
    return self._dlltype(name)
  File "/usr/lib/python3.8/ctypes/__init__.py", line 373, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: /usr/local/lib/libIOTCAPIs_ALL.dylib: cannot open shared object file: No such file or directory
```

### Environment

* OS: [e.g. Linux / Windows / macOS] Linux
* Python version, get it with: Python 3.8.5

```bash
python --version
```

**Comment by kroo** [comment]
Thanks—this is definitely what happens when you don’t test enough on other platforms before launching :).  Should be simple enough to fix.

**Comment by krystiancharubin** [comment]
I've actually got it running on Windows using WSL, just needed to install Xming on windows to get video window working.

**Comment by kroo** [comment]
Do you mind contributing some documentation on how to get it running on wsl?

**Comment by krystiancharubin** [comment]
Sure, I'll put something together over the weeked.

**Comment by kroo** [comment]
Resolved in v1.1.0
